auth_service/user_management/claims_repository/core/seed.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+Functions to seed the repository with user claims.
+"""
+
+import logging
+
+from hexkit.protocols.dao import MultipleHitsFoundError, NoHitsFoundError
+
+from auth_service.config import Config
+from auth_service.deps import get_mongodb_config, get_mongodb_dao_factory
+from auth_service.user_management.claims_repository.core.claims import (
+    create_data_steward_claim,
+)
+from auth_service.user_management.claims_repository.core.utils import is_data_steward
+from auth_service.user_management.claims_repository.deps import (
+    get_claim_dao,
+    get_claim_dao_factory,
+    get_claim_dao_factory_config,
+)
+from auth_service.user_management.user_registry.deps import (
+    get_user_dao,
+    get_user_dao_factory,
+    get_user_dao_factory_config,
+)
+
+__all__ = ["seed_claims"]
+
+log = logging.getLogger(__name__)
+
+
+async def seed_claims(config: Config) -> None:
+    """Seed the claims repository with claims specified in the config."""
```

Comment:
Shouldn't we be  a bit more specific:
```suggestion
async def seed_data_steward_claims(config: Config) -> None:
    """Seed the claims repository with claims for the data steward role specified in the config."""
```

Reply:
Yes, that's better. If other claims will be added later, this should be handled by another function.